src/history/history.rs

```diff
@@ -106,7 +106,6 @@ use super::file::time_to_seconds;
 const DFLT_FISH_HISTORY_SESSION_ID: &wstr = L!("fish");
 
 /// When we rewrite the history, the number of items we keep.
-// FIXME: https://github.com/rust-lang/rust/issues/67441
```

Comment:
> This comment should have been removed in 3736636d999d66d5452e28c4962b8f3c259386b8
 > ("msrv: update to Rust 1.85", 2025-10-19)

FWIW there is  `git log --format=reference`,
(also some Git UIs might have their own shortcut for copying this)